### PR TITLE
Add more authorization query options

### PR DESCRIPTION
**Pull request by omer9564**

### CODE REVIEW

File: pkg/enforcement/all_tenants.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+func (e *PermitEnforcer) getAllTenantsCheckEndpoint() string {
+	return e.getEndpointByPolicyPackage(allTenantsPolicyPackage)
+}
+
+func (e *PermitEnforcer) parseAllTenantsResponse(res *http.Response) ([]AllTenantsCheckResponse, error) {
+	var result []AllTenantsCheckResponse
+	bodyBytes, err := io.ReadAll(res.Body)
+	if err != nil {
+		permitError := errors.NewPermitUnexpectedError(err, res)
+		e.logger.Error("error reading Permit.Bulk() response from PDP", zap.Error(permitError))
```
Comment (RazcoDev) [inline pkg/enforcement/all_tenants.go:39]: fix error - not permit.bulk

File: pkg/enforcement/all_tenants.go [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+	if e.config.GetOpaUrl() != "" {
+		opaStruct := &opaAllTenantsResponse{
+			Result: &allowedTenantsResponse{
+				&result,
+			},
+		}
+
+		if err := json.Unmarshal(bodyBytes, opaStruct); err != nil {
+			permitError := errors.NewPermitUnexpectedError(err, res)
+			e.logger.Error("error unmarshalling Permit.BulkCheck() response from OPA", zap.Error(permitError))
```
Comment (RazcoDev) [inline pkg/enforcement/all_tenants.go:56]: same here

File: pkg/enforcement/all_tenants.go [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+			return nil, err
+		}
+	} else {
+		pdpStruct := &allowedTenantsResponse{&result}
+		if err := json.Unmarshal(bodyBytes, &pdpStruct); err != nil {
+			permitError := errors.NewPermitUnexpectedError(err, res)
+			e.logger.Error("error unmarshalling Permit.BulkCheck response from PDP", zap.Error(permitError))
```
Comment (RazcoDev) [inline pkg/enforcement/all_tenants.go:63]: and here

**Review by RazcoDev — APPROVED** [review]

File: pkg/enforcement/all_tenants.go (hunk 1 above)
Comment (omer9564) [reply pkg/enforcement/all_tenants.go]: fixed

File: pkg/enforcement/all_tenants.go (hunk 2 above)
Comment (omer9564) [reply pkg/enforcement/all_tenants.go]: fixed

File: pkg/enforcement/all_tenants.go (hunk 3 above)
Comment (omer9564) [reply pkg/enforcement/all_tenants.go]: fixed

**Pull request merged**